posts/2021/04/spot-the-differences.md

```diff
@@ -0,0 +1,213 @@
+<!--
+.. title: Spot the differences: what is new in Spyder 5?
+.. slug: spot-the-diffenrences
+.. date: 2021-04-16 08:00:00 UTC-06:00
+.. author: Isabela Presedo-Floyd and Juanita Gomez
+.. tags: Spyder, UX/UI, release
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+![Spyder 5 versus Spyder 4](/images/spyder5-header.png)
+
+*This blog post is a conversation. Portions lead by Juanita Gomez are marked 
+as **Juanita**, and those lead by Isabela Presedo-Floyd are marked as 
+**Isabela**.*
+
+## What did we do?*
+
+**[Juanita]** [Spyder](https://www.spyder-ide.org/) was created 10 years ago and it has had the 
+contributions of a great number of developers who have written code, proposed 
+ideas, opened issues and tested PRs in order to build a piece of Spyder on 
+their own. We (Spyder’s team) have been lucky to have such a great community 
+of people contributing, but this is the first time that we decided to ask for 
+help from an UX/UI expert! Having the help of this great amount of people has 
+resulted in inconsistencies around Spyder’s interface which we didn’t stop to 
+analyze until now. 
+
+When Isabela joined Quansight, we realized that we had an opportunity of 
+improving Spyder’s interface with her help. We thought her skill set was 
+everything we needed to make Spyder’s UI better. We conducted a community 
```

Comment:
```suggestion
everything we needed to make Spyder’s UI better. So we started by conducting a community 
```

Reply:
I think the survey was not actually a consequence of this, so this is not quite accurate but if it helps the flow of the post I can make the change. Thoughts @isabela-pf ?

Reply:
I think it can be accurate and flow! Maybe `So we started by reviewing the results of a community survey from a few months ago and realized…`

Reply:
My bad - but the current way it is it seems to bounce from one idea to another seemingly unrelated. I like @isabela-pf 's proposal